### ISSUE TITLE
Checking similarity of input ligand w.r.t training set

### ISSUE THREAD

**Issue by zhang-ivy** [jwohlwend/boltz#234]
As described in the preprint, one of the criteria for constructing the validation set is "at least one of the small-molecules exhibits a tanimoto similarity of 0.8 or less to any small-molecule in the training set." Would it be possible to share your workflow (code and ligands in training set) for computing small molecule similarity to the training set? I am curious how similar my input ligand is to the training set ligands.
